pvm.py

```diff
@@ -21,6 +21,8 @@ def _init_logger():
 # _logger.info('App started in %s', os.getcwd())
 # _logger.debug('App started in %s', os.getcwd())
 
+# Place your videos in this folder for autostart
+PEFIX_PATH = "/home/pi/Videos/"
```

Comment:
missing instructions about this on the readme!

Reply:
Added Videos section in README